verifiable-db/src/row_tree/api.rs

```diff
@@ -179,23 +181,48 @@ pub enum CircuitInput {
 
 impl CircuitInput {
     pub fn leaf(identifier: u64, value: U256, cells_proof: Vec<u8>) -> Result<Self> {
-        let circuit = LeafCircuit::new(IndexTuple::new(F::from_canonical_u64(identifier), value));
+        Self::leaf_multiplier(identifier, value, false, cells_proof)
+    }
+    pub fn leaf_multiplier(
+        identifier: u64,
+        value: U256,
+        is_multiplier: bool,
```

Comment:
Same comment as for cells tree API: I would expect this function doesn't have an `is_multiplier` flag as input, but will instead be called only when `is_multiplier = true`